EnglishTranslation-master/ITEM.tsv

```diff
@@ -9818,18 +9818,18 @@ ITEM_20151224_009817	Spell Amplifier
 ITEM_20151224_009818	An amplifier carried by Black Maizes. An item wanted by Revelator Mihail.
 ITEM_20151224_009819	Raffly Soul Crystal
 ITEM_20151224_009820	Vubbe Warrior Mind Crystal
-ITEM_20160224_009821	Many followed suit once one of the heirs of the more prestigious families fashioned their hair this way. It must be said however, that it is difficult to diverge all of its charm without the person being attractive in the first place.
-ITEM_20160224_009822	A hairstyle restored around the time of the Millenary which many people tried to imitate. However, it was mostly done by the nobility because it is difficult to cut and maintain.
+ITEM_20160224_009821	Once a heir of one the more prestigious families fashioned their hair this way, many followed suit. It must be said however, it's difficult to capitalize on its charm without being attractive in the first place.
```

Comment:
I'll probably change this back to "Once the heir," whenever I go in for touchups.